flexget/components/notify/notifiers/mqtt.py

```diff
@@ -0,0 +1,204 @@
+                    self.config.get('certificates',{}).update({'broker_ca_cert': blank2None(self.config.get('certificates',{}).get('broker_ca_cert'))})
+                    self.config.get('certificates',{}).update({'client_cert': blank2None(self.config.get('certificates',{}).get('client_cert'))})
+                    self.config.get('certificates',{}).update({'client_key': blank2None(self.config.get('certificates',{}).get('client_key'))})
+                    self.config.get('certificates',{}).update({'tls_version': blank2None(self.config.get('certificates',{}).get('tls_version'))})
+                    logger.trace('MQTT notify config={}'.format(str(self.config)))
+
+                    mqtt.Client.__init__(self, protocol=PublishMQTT.MQTT_proto_map.get(self.config.get('broker_protocol',mqtt.MQTTv311)), transport=self.config.get('broker_transport', 'tcp') )
+
+                    self.enable_logger(logger=logger)
+
+                    self.on_log = PublishMQTT.on_log_cb
+                    self.on_publish = PublishMQTT.on_publish_cb
+                    self.on_disconnect = PublishMQTT.on_disconnect_cb
+
+                    #Handle SSL/TLS communication w/out certificate authentication
+                    if not self.config.get('certificates',{}).get('client_cert',False) and self.config.get('enable_encrypted_communication',False):
+                        self.tls_set(ca_certs=certs.get('broker_ca_cert',None),
+                                      certfile=None,
+                                      keyfile=None,
+                                      cert_reqs=ssl.CERT_NONE)
+
+                        self.tls_insecure_set(True)
+
+                        logger.verbose('Basic SSL/TLS encrypted communications enabled')
+                        logger.verbose('TLS insecure cert mode enabled. Broker cert will not be validated')
+
+                    #Handle SSL/TLS communication with certificate authentication
+                    if self.config.get('certificates',False):
+                        certs = self.config.get('certificates',{})
+                        logger.debug('TLS certificate config: {}'.format(str(certs)))
+
+                        tls_version_map = {'tlsv1.2': ssl.PROTOCOL_TLSv1_2, 'tlsv1.1': ssl.PROTOCOL_TLSv1_1, 'tlsv1': ssl.PROTOCOL_TLSv1, '': None}
+                        tls_version = tls_version_map.get(certs.get('tls_version'),ssl.PROTOCOL_TLSv1_2)
+                        logger.verbose('TLS version is {}'.format(str(tls_version)))
+
+                        if certs.get('validate_broker_cert',True):
+                            cert_required = ssl.CERT_REQUIRED
+                        else:
+                            cert_required = ssl.CERT_NONE
+                        logger.verbose('TLS cert_required={}'.format(str(cert_required)))
+
+                        self.tls_set(ca_certs=certs.get('broker_ca_cert',None),
```

Comment:
no need to explicitly use `None` as default for dict.get(), it's the default

Reply:
Done with commit b90192ccf26b5cfb5843bdd1da8cebe81abd4d3a